src/bin/load_test.rs

```diff
@@ -147,20 +160,9 @@ async fn main() {
             .await;
     }
 
-    // Refresh wallet balance
+    // Refresh wallet core balance
```

Comment:
Consider extracting wallet and identity preparation to a separate function (for example lines 149-227), to avoid building large functions that do multiple things.

Reply:
This code is not meant to stay. It should be replaced by strategies.